crates/cast/tests/cli/main.rs

```diff
@@ -818,7 +818,7 @@ type                 2
 blobGasPrice         
 blobGasUsed          
 to                   0x68b3465833fb72A70ecDF485E0e4C7bD8665Fc45
-revertReason         Transaction too old, data: "0x08c379a0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000135472616e73616374696f6e20746f6f206f6c6400000000000000000000000000"
+revertReason         revert: Transaction too old, data: "0x08c379a0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000135472616e73616374696f6e20746f6f206f6c6400000000000000000000000000"
```

Comment:
maybe we should redact this [..] I notice that's flaky. there's another one I am currently looking into with different order of json fields and fails CI, not related